webClient/src/app/app.component.html

```diff
@@ -26,53 +28,60 @@
+        i18n="action mode|how to handle launch depending on whether target app has already been launched@@app-target-mode">App Target Mode: </label>
+        <input type="radio" [(ngModel)]="targetMode" name="targetMode" value="PluginCreate">
+        <label for="targetCreate" i18n="create new instance@@create-new">Create New</label>
 
-        <input type="radio" [(ngModel)]="targetMode" value="PluginFindAnyOrCreate">
-        <label for="targetReuse">Reuse Any Open</label>
+        <input type="radio" [(ngModel)]="targetMode" name="targetMode" value="PluginFindAnyOrCreate">
+        <label for="targetReuse" i18n="reuse any open app@@reuse-any-open">Reuse Any Open</label>
       </div>      
-      <span class="bold-text hide-it">Target App Window Title (this will become the title for the target window)</span>
+      <span class="bold-text hide-it" i18n="window title for target app@@target-app-window-title">
+        Target App Window Title (this will become the title for the target window)</span>
       <div class="div-input hide-it">
         <input class="iframe-input input-height input-corner input-text shadowed" type="text" name="appWindowTitle" id="appWindowTitle" placeholder="app window title" value="My New Title"/>
       </div>
-      <span class="div-input bold-text">Parameters:</span>
+      <span class="div-input bold-text" i18n="parameters@@parameters">Parameters:</span>
       <div class="div-textarea-input">
         <!-- The text here is merely an example which provides some connection details for the terminal app. It could be anything so long as the receiving App supports it.
              In this example App, the contents here will be put inside of a JSON with the contents as the "data" attribute. -->
-        <textarea class="iframe-input input-corner input-text shadowed" rows="10" cols="50" [(ngModel)]="requestText" ></textarea>
+        <textarea class="iframe-input input-corner input-text shadowed" rows="10" cols="50" [(ngModel)]="parameters" ></textarea>
       </div>
       <div style="width: 100%">
-        <span>App Status or Message:</span>
+        <span i18n="status of app or message from app@@app-status">App Status or Message:</span>
         <p class="display-text shadowed disable-effect" id="status">{{callStatus}}</p>
       </div>
     </div>
   </div>
   <div class="test-panel dataservice-test-panel">
     <div class="bottom-10">
-      <span class="bigger-bold-text">Dataservice Request Test</span>
+      <span class="bigger-bold-text" i18n="test of dataservice requests@@dataservice-request-test">Dataservice Request Test</span>
     </div>
     <div>
-      <input placeholder="Message" [(ngModel)]="helloText"
+      <input placeholder="Message" 
+             i18n-placeholder="message|indicates that the user will enter a message@@appComponentMessage" [(ngModel)]="helloText"
       (keyup.enter)="sayHello()"/>
-      <button (click)="sayHello()">Run</button>
+      <button (click)="sayHello()"
```

Comment:
This doesn't have the zlux-button code that is in https://github.com/zowe/sample-app/commits/lab/step-4-widgets-complete,